<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>注册账号</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 255, 255);
border-radius:15px;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;楷体&quot;;</string>
   </property>
   <property name="text">
    <string>注册账号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>300</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;楷体&quot;;</string>
   </property>
   <property name="text">
    <string>工作人员注册时请填入校验码，非工作人员无需填写</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>331</width>
     <height>81</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="submit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:#03a9f4;
color:#fff;
border-radius:15px;
}</string>
    </property>
    <property name="text">
     <string>提交</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:#03a9f4;
color:#fff;
border-radius:15px;
}</string>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>401</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;华文行楷&quot;;
font: 9pt &quot;楷体&quot;;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="enter_name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>     输入姓名：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:15px;
background-color: rgb(255, 255, 255);
color:#03a9f4;</string>
    </property>
    <property name="placeholderText">
     <string>请输入姓名</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="id">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:15px;
background-color: rgb(255, 255, 255);
color:#03a9f4;</string>
    </property>
    <property name="placeholderText">
     <string>请输入账号（身份证号</string>
    </property>
   </widget>
   <widget class="QLabel" name="passNumber1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>90</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>     输入密码：</string>
    </property>
   </widget>
   <widget class="QLabel" name="passNumber2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>90</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string> 再次输入密码：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:15px;
background-color: rgb(255, 255, 255);
color:#03a9f4;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>请输入密码</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:15px;
background-color: rgb(255, 255, 255);
color:#03a9f4;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>请再次输入密码</string>
    </property>
   </widget>
   <widget class="QLabel" name="check">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>       校验码：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="checkCode">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>210</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:15px;
background-color: rgb(255, 255, 255);
color:#03a9f4;</string>
    </property>
    <property name="placeholderText">
     <string>工作人员注册时输入</string>
    </property>
   </widget>
   <widget class="QLabel" name="account">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>     输入账号：</string>
    </property>
   </widget>
   <widget class="QLabel" name="sex_name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>90</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>         性别：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="select">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:15px;
color: rgb(85, 255, 255);
background-color: rgb(255, 255, 255);</string>
    </property>
    <item>
     <property name="text">
      <string>男</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>女</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>保密</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
